vector/v.distance/distance.c

```diff
@@ -12,7 +12,7 @@ int get_line_box(const struct line_pnts *Points,
     int i;
 
     if (Points->n_points == 0) {
-	box->E = box->W = box->N = box->S = box->T = box->B = 0.0 / 0.0;
+	box->E = box->W = box->N = box->S = box->T = box->B = INFINITY;
```

Comment:
This change is not Cmake related and should be separated out

(into a new INFINITY related PR)